src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -257,7 +257,8 @@ public function getLimit()
         $limit = null;
 
         if ($this->isLimited()) {
-            $limit = (int) $this->parameters->get('limit', 10);
+            $limit= $this->parameters->get('limit', 10);
```

Comment:
@aRn0D fixing it at moment :wink: 
@pamil I aggree about return abrupt, but not about ternary operator, it will create a noise and this logic already happens inside `parameterBag->get` and `request->get`, so, it is not necessary happen here